### DISCUSSION TITLE
Aris Studio Development - Dynamically create Select/Choice button at runtime

### DISCUSSION THREAD

**Discussion by kiraio-moe** [Tualin14/baPlayer#22] [Development]
Hi @Tualin14, I'm wondering why you chose to only use a fixed number of select/choice buttons (up to 3 buttons) instead of creating them at runtime? Is it because the placement of the buttons must be accurate like in the game or is there something else? Thanks.

**Comment by ttttttttttttttllllllllllllll** [comment]
Just to be consistent with the game location, the automatically generated location will be uncertain and there is no greater than three button in the game.

**Reply by kiraio-moe** [reply]
Thanks for confirming, I was intending to generate them at runtime in case the user wanted to create more than 3 buttons.

**Reply by ttttttttttttttllllllllllllll** [reply]
I think it would be better to recover the game here. The user should emulate the game to split the text to assign button content.
